pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/customer-admin-reconciler.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: customer-admin-reconciler
+rules:
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - rolebindings
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ''
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+  - create
```

Comment:
This does not sound right.

Reply:
@kwoodson can you provide the value of `/tmp/rolebindings` and `oc get clusterrole.rbac -o yaml` - something seems off in your environment.

Reply:
this is the contents of the configmap:
```
    kind: List
    apiVersion: v1
    metadata: {}
    items:
    - apiVersion: rbac.authorization.k8s.io/v1
      kind: RoleBinding
      metadata:
        name: customer-admin
      roleRef:
        apiGroup: rbac.authorization.k8s.io
        kind: ClusterRole
        name: admin
      subjects:
      - apiGroup: rbac.authorization.k8s.io
        kind: Group
        name: customer-admins
    - apiVersion: rbac.authorization.k8s.io/v1
      kind: RoleBinding
      metadata:
        name: customer-admin-project
      roleRef:
        apiGroup: rbac.authorization.k8s.io
        kind: ClusterRole
        name: customer-admin-project
      subjects:
      - apiGroup: rbac.authorization.k8s.io
        kind: Group
        name: customer-admins
    - apiVersion: rbac.authorization.k8s.io/v1
      kind: RoleBinding
      metadata:
        name: customer-view
      roleRef:
        apiGroup: rbac.authorization.k8s.io
        kind: ClusterRole
        name: view
      subjects:
      - apiGroup: rbac.authorization.k8s.io
        kind: Group
        name: customer-readers
```

```
[root@master-000000 ~]# oc get clusterrole.rbac customer-admin-reconciler -o yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  creationTimestamp: 2018-10-23T15:51:39Z
  name: customer-admin-reconciler
  resourceVersion: "359445"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterroles/customer-admin-reconciler
  uid: 87ca906f-d6db-11e8-98ce-000d3a4fe942
rules:
- apiGroups:
  - rbac.authorization.k8s.io
  resources:
  - rolebindings
  verbs:
  - create
  - delete
  - get
  - list
  - patch
  - update
  - watch
- apiGroups:
  - ""
  resources:
  - namespaces
  verbs:
  - get
  - list
  - watch
```

Reply:
@enj @kwoodson any further thoughts on this?